### PR TITLE
Re-enable depth buffer on render target

### LINKED ISSUE #55: THREE.DoubleSide rendering incorrectly
Since updating to version 0.2 THREE.DoubleSide seems to be rendering incorrectly in VR mode for me (on cardboard), the BackSide will often be rendered in front of the FrontSide. It's inconsistent. Is this happening with anyone else?

### PR DESCRIPTION
**Pull request by njam**
This reverts commit 5c6f9d789689d1b6eba0e09151de8fc265c5f0a3.

Fixes #55
Reverts #35

It's not a good idea, because it disables depth buffer in the original scene.

@zazabe @mrdoob fyi

### CODE REVIEW

File: src/cardboard-distorter.js [hunk 1]
```diff
@@ -98,8 +98,7 @@ function createRenderTarget(renderer) {
   var parameters = {minFilter: THREE.LinearFilter,
                     magFilter: THREE.LinearFilter,
                     format: THREE.RGBFormat,
-                    stencilBuffer: false,
-                    depthBuffer: false};
```
Comment (mrdoob) [inline src/cardboard-distorter.js:102]: I think we can keep this one?

Comment (borismus) [inline src/cardboard-distorter.js:102]: Ah, true. Although this distortion code is going to need to change very soon.

Comment (mrdoob) [inline src/cardboard-distorter.js:102]: Yes. As far as I understand, cardboard-distorter basically distorts a picture, there is no depthBuffer usage.

Comment (idlewan) [inline src/cardboard-distorter.js:102]: There isn't a need for depthBuffer for the distortion, however it is needed for that [textureTarget](https://github.com/borismus/webvr-boilerplate/blob/32cd6f5/src/cardboard-distorter.js#L135).

Unless I missed something, the `createRenterTarget` function is currently used for the `genuineRender` target (the original render function renders to it when in VR mode), so it does create problems if it doesn't have a depth buffer. The ShaderPass then renders using the same `genuineRender` function but using the default target ([cardboard-distorter.js#L168](https://github.com/borismus/webvr-boilerplate/blob/32cd6f5/src/cardboard-distorter.js#L168) and [#L91](https://github.com/borismus/webvr-boilerplate/blob/32cd6f5/src/cardboard-distorter.js#L91)).

I just encountered that problem, where the non-cardboard view renders fine, but as soon as you switch to cardboard view some objects that should be behind others appear in front (issue #55 is very related).
Removing the `depthBuffer: false` solves it.

So https://github.com/borismus/webvr-boilerplate/commit/32cd6f543534960a104e2df9d247c451b74f9e01 actually re-introduce the issue, and #55 isn't fixed as a result.

The distorter doesn't need a depthBuffer, but that `textureTarget` is used as output for genuine rendering and input for the distortion. It does need a depthBuffer, otherwise the genuine rendering doesn't render properly.
Can we revert https://github.com/borismus/webvr-boilerplate/commit/32cd6f543534960a104e2df9d247c451b74f9e01 please?

Comment (mrdoob) [inline src/cardboard-distorter.js:102]: @idlewan hard to follow... would you be up for doing a PR with the fix?

Comment (idlewan) [inline src/cardboard-distorter.js:102]: Same fix for #57 and #55, just remove the `depthBuffer: false`, i.e. revert https://github.com/borismus/webvr-boilerplate/commit/32cd6f543534960a104e2df9d247c451b74f9e01

Comment (borismus) [inline src/cardboard-distorter.js:102]: Fixed in https://github.com/borismus/webvr-boilerplate/commit/f48f016dee7833fcf1522157e77913167f16cd7a
